tests/FakeObject.php

```diff
@@ -1,7 +1,9 @@
 <?php
 namespace Aura\Sql;
 
-class FakeObject
+use stdClass;
```

Comment:
Not sure why you want to extend stdClass.

Reply:
Extending stdClass avoids the "Deprecate dynamic properties" error.

https://wiki.php.net/rfc/deprecate_dynamic_properties

Reply:
But in this case we don't have dynamic properties right ? We already have defined the property. Did I missed something ?

Reply:
Confirmed. We missed the id and name properties there.